File: code/listen.java
java.util.List<String> anListWithTheNameList = java.util.Arrays.asList("catfish", "water", "tulips", "blue paint");
anListWithTheNameList.set(1, "bottle of water");
for (String s : anListWithTheNameList) {
    System.out.println(s);
}
// Ausgabe:
// catfish
// bottle of water
// tulips
// blue paint

File: code/listen.kt
val anListWithTheNameList = mutableListOf("catfish", "water", "tulips", "blue paint")
anListWithTheNameList[1] = "bottle of water"
for (s in anListWithTheNameList) {
    println(s)
}
// Ausgabe:
// catfish
// bottle of water
// tulips
// blue paint